client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -84,6 +84,15 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 		setDetailsExpanded( Object.fromEntries( detailsExpandedAsArray ) );
 	};
 
+	const setIntervalLength = ( interval: IntervalLength ) => {
+		if ( intervalLength === IntervalLength.ANNUALLY ) {
+			setDetailsExpanded( { google: false, titan: true } );
+		}
+		setIntervalLengthPure( interval );
```

Comment:
I'm not a big fan of this `Pure` suffix, as it doesn't convey any meaning. What do you think of the following instead?

```suggestion
	const handleIntervalChange = ( interval: IntervalLength ) => {
		if ( intervalLength === IntervalLength.ANNUALLY ) {
			setDetailsExpanded( { google: false, titan: true } );
		}

		setIntervalLength( interval );
```